### PR TITLE
Update plugin to v2.0 of the api

### PR DESCRIPTION
**Pull request by ph**

### CODE REVIEW

File: lib/logstash/inputs/rabbitmq.rb [hunk 1]
```diff
@@ -250,8 +250,13 @@ def internal_queue_consume!
           @codec.decode(data) do |event|
             decorate(event)
             if @metadata_enabled
-              event["@metadata"]["rabbitmq_headers"] = get_headers(metadata)
-              event["@metadata"]["rabbitmq_properties"] = get_properties(metadata)
+              meta = {
+                "rabbitmq_headers" => get_headers(metadata),
+                "rabbitmq_properties" => get_properties(metadata)
+
```
Comment (talevy) [inline lib/logstash/inputs/rabbitmq.rb]: new lines?

**Comment by ph** [comment]
@talevy updated with changes

**Comment by talevy** [comment]
@ph I don't see the changes

**Comment by ph** [comment]
pushed to my own remote instead of this plugin :(

**Comment by talevy** [comment]
figured. LGTM

**Pull request merged**